python/pyarrow/tests/test_pandas.py

```diff
@@ -647,6 +647,23 @@ def test_mismatch_metadata_schema(self):
             result = new_table.to_pandas()
             tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize("index", ["a", ["a", "b"]])
+    def test_to_pandas_types_mapper_index(self, index):
```

Comment:
The other types_mapper related tests are lower in the file (not in one of the classes), so I would move it there (eg just below `test_roundtrip_empty_table_with_extension_dtype_index`)

(Sidenote, seeing that `test_roundtrip_empty_table_with_extension_dtype_index` test, that seems to be something that should be fixed as well, preserving interval index)

Reply:
Moved